style.css

```diff
@@ -0,0 +1,1016 @@
+  line-height: 24px;
+  color: #344563;
+  margin-top: 30px;
+}
+
+.newme-description-tonic {
+  width: 295px;
+  height: 76px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0;
+  width: 203px;
+  height: 24px;
+  font-family: normal;
+  color: #6070ff;
+}
+
+.languages-tonic {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+  top: 180px;
+  margin-top: -10px;
+  margin-bottom: 10px;
+  font-family: normal;
+  color: #6070ff;
+}
+
+.language1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 12px;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin: 0 5px;
+  border: 0;
+}
+
+.language1 p {
+  width: 29px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  top: 4px;
+  left: 12px;
+}
+
+.language2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  border: 0;
+  left: 61px;
+  margin: 0 5px;
+}
+
+.language2 p {
+  width: 21px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  top: 4px;
+  left: 12px;
+}
+
+.language3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  border: 0;
+  left: 114px;
+  margin: 0 5px;
+}
+
+.language3 p {
+  width: 65px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  top: 4px;
+  left: 12px;
+}
+
+.newmes {
+  width: 295px;
+  height: 72px;
+}
+
+.newmes-tonic {
+  width: 295px;
+  height: 72px;
+  margin-top: -5px;
+}
+
+.go-newmes {
+  box-sizing: border-box;
+  align-items: center;
+  padding: 12px;
+  width: 140px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  color: #396df2;
+  transition: transform 0.4s ease-in-out;
+}
+
+.go-newmes:hover {
+  transform: scale(1.1);
+  background-color: rgb(19, 4, 72);
+  color: rgb(223, 225, 230);
+}
```

Comment:
- [ ] kindly consider sticking to the Figma design as much as possible to make your project a duplicate of the given design  in order to achieve that consider giving your button's hover effect one of the colors below as given in the Figma design 
- your design 👇 
<img width="375" alt="Screen Shot 2023-05-05 at 19 59 32" src="https://user-images.githubusercontent.com/60146030/236533497-b74f1d49-db08-4880-a493-97cf05cc50ab.png">
 

- Figma design 👇 
<img width="528" alt="Screen Shot 2023-05-05 at 20 00 12" src="https://user-images.githubusercontent.com/60146030/236533660-2563200c-0ade-4548-8bbf-aa92a909ffef.png">

Reply:
Ok. I delete a background color. So it is working without blue color.

![image](https://user-images.githubusercontent.com/35693330/236589729-9c7e1053-8945-4557-97a2-55be80570c0f.png)